app/controllers/api/v1/road_trip_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::RoadTripController < ApplicationController
+  def show
```

Comment:
Big next goal is to push alot  of this functionality down away from the controller.